lte/gateway/python/scripts/BUILD.bazel

```diff
@@ -532,3 +532,10 @@ py_binary(
     legacy_create_init = False,
     tags = TAG_UTIL_SCRIPT,
 )
+
+py_binary(
+    name = "dhcp_helper_cli",
```

Comment:
Should this be available in a debian artifact? Then it needs to be added to the SCRIPTS list at the beginning of the file.

Reply:
This might be part of https://github.com/magma/magma/issues/14459.

But we will need to take care how we merge these issues (use the script in pipelined only after it's clear how it is deployed).

Reply:
Then fyi, adding it to setup.py will iiuc already include it in the make created debian artifact.

Reply:
Oh, that's most likely not what we want.